### ISSUE TITLE
DriverManager appears to be out of step with the latest Edge implementation and Selenium4.0.0 alpha4

### ISSUE THREAD

**Issue by aonorgate** [rosolko/WebDriverManager.Net#62]
The new Edge post build 80 requires use of a different driver initialization and has been optimized for Selenium 4.0.0alpha04. 

I don't seem to be able to get Driver Manager to work with the current Edge build and suspect it may be due to these recent changes which Microsoft declares are breaking to the previous method of initializing the webdriver and browser.

**Comment by rosolko** [comment]
Got it. 
Looks like I don't catch the problem.

As I understood, the latest download version of edge driver is incompatible with latest edge browser, I am right?

If yes, just pre-configure edge config to get compatible version of driver.
If no, please, describe a problem more specifically.

Thanks!

**Comment by aonorgate** [comment]
So, using Edge Chromium 80, Legacy Edge 18362 and Selenium 4.0.0_alpha04:

LegacyEdgeConfig does not fully work to drive the Legacy Edge browser. The following code will Launch the Browser but cannot accept commands:
```
var options = new EdgeOptions();
            new DriverManager().SetUpDriver(new LegacyEdgeConfig());
            EdgeDriver Driver = new EdgeDriver(options);
```

If i use EdgeConfig it works correctly for the Legacy Edge browser:
```
            var options = new EdgeOptions();
            new DriverManager().SetUpDriver(new EdgeConfig());
            EdgeDriver Driver = new EdgeDriver(options);
```

Following the new initialization process for build 80 (as per here: https://docs.microsoft.com/en-us/microsoft-edge/webdriver-chromium) I would expect this code to work but instead it just launches the Legacy Edge browser:
```
            var options = new EdgeOptions(false);
            options.BinaryLocation = @"C:\Program Files (x86)\Microsoft\Edge Beta\Application\msedge.exe";
            new DriverManager().SetUpDriver(new EdgeConfig());
            EdgeDriver Driver = new EdgeDriver(options);
```

It looks to be that the recent changes to the Webdriver by MS have not been incorporated into DriverManager

**Comment by rosolko** [comment]
If use latest stable 3 version of selenium not an alpha one what will be result?

**Comment by aonorgate** [comment]
With latest stable 3.141:

LegacyEdgeConfig - Launches legacy browser but unable to command - same behaviour            
```
            var options = new EdgeOptions();
            new DriverManager().SetUpDriver(new LegacyEdgeConfig());
            EdgeDriver Driver = new EdgeDriver(options);
```

EdgeConfig - Launches legacy browser and can command - same behaviour
```
            var options = new EdgeOptions();
            new DriverManager().SetUpDriver(new EdgeConfig());
            EdgeDriver Driver = new EdgeDriver(options);
```

The final code snippet form above is not supported in Selenium 3, MS has an alternative but strongly pushes using the Alpha releases. If I use the other approach it initialises a RemoteWebDriver as as such has no need for DriverManager.

So with the latest stable release Edge Chromium build 80 cannot be used with DriverManager due to the manner in which it must now be launched using the latest EdgeOptions or via legacy code and a RemoteWebDriver.

Also LegacyEdgeConfig doesn't appear to work, I am unsure if this is new with build 80 of Edge as i have only recently begun using DriverManager.

**Comment by aonorgate** [comment]
What may also be complicating things is that from v18 of the Legacy Edge WebDriver onwards, no exe is provided from download and the WebDriver is a developer feature of Windows10. 
See here: https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/

I have found it is still possible to locate the WebDriver exe locally once the feature has been enabled but I suspect DriverManager will now be stuck using v17.

The Edge Chromium webdrivers remain available for download.

**Comment by rosolko** [comment]
Update edge config to use latest version from a site provided before.
Hope it will fix the problem.
Also closes #64
